lib/webdriveragent.js

```diff
@@ -322,6 +322,15 @@ class WebDriverAgent {
     }
   }
 
+  async getStartTime () {
+    let startTime = new Date();
+    if (this.realDevice) {
+      let {stdout} = await exec('idevicedate');
```

Comment:
what if there are multiple devices connected to the machine? can you explicitly pass the uuid of the target device to this command?